.github/workflows/test.yaml

```diff
@@ -240,6 +240,7 @@ jobs:
             'mistral1',
             'notdiamond',
             'openai',
+            'pandas',
```

Comment:
I would call this `pandas-test` to show that it's just for testing, not a user-facing extra